package.json

```diff
@@ -15,8 +15,8 @@
     "build": "node scripts/build",
     "build-test-binary": "cd test/binary && node-gyp rebuild && cp build/Release/hello.node ../integration/hello.node",
     "codecov": "codecov",
-    "test": "node --expose-gc --max_old_space_size=3072 node_modules/.bin/jest",
-    "test-coverage": "node --expose-gc --max_old_space_size=3072 node_modules/.bin/jest --coverage --globals \"{\\\"coverage\\\":true}\" && codecov",
```

Comment:
`--max_old_space_size` is necessary here.

Reply:
Thanks! I added cross-spawn and that got past the memory issue, but now jest doesn't find any tests on Windows 🤔